frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Card, _Image, Button, Row, Col} from "react-bootstrap";
 // import cowHead from "../../../CowHead.png"; 
```

Comment:
small nitpick: if this comment isn't needed anymore, it would be great if we could remove it!
```suggestion
```